docs/platform/block-storage/block-storage-use-cases/index.md

```diff
@@ -8,58 +8,80 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
 published: 2019-08-27
 modified_by:
   name: Linode
-title: "Block Storage Use Cases"
+title: "Use Cases for Block Storage"
 contributor:
   name: Linode
 ---
 
```

Comment:
I replaced "server" with "instance" or "cloud instance" throughout the guide to match the newer marketing-speak for how we talk about Linodes. Also stripped out some references to Linode where the information was universally applicable to Block Storage (mostly this was in the intro sections)